cache/lru/lru.go

```diff
@@ -323,6 +340,13 @@ func (tc *TieredLRUCache) resize() {
 // remove the elements with time inbetween removes,
 // but only if they are not in the cache at the time of removal
 func (tc *TieredLRUCache) throttledRemove(indexes []types.ObjectIndex) {
+	defer func() {
+		if msg := recover(); msg != nil {
+			tc.logger.Errorf(
```

Comment:
Is printing stack trace here possible? The stack trace which lead to the problem, not the trace at this point in the recover function.